CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Unreleased
 ------
+* [#965](https://github.com/Shopify/shopify-app-cli/pull/965): Remove --no-optional when using npm to create new extension
```

Comment:
I think it would make sense to update this text. You haven't changed it just for extensions but for all projects in the CLI with NPM dependencies.

Reply:
Updated!